# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1457642292 18000
#      Thu Mar 10 15:38:12 2016 -0500
# Branch stable
# Node ID 2b66f30a0de7a09801c2a3566021cde7e194d3d3
# Parent  7bb71bf2f30f58080c7c29ce30dad092d9644c21
maint: Bump version number for 4.0.1-rc4.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -14,24 +14,24 @@ dnl Copyright (C) 1993-2015 John W. Eato
 ### FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
 ### for more details.
 ###
 ### You should have received a copy of the GNU General Public License
 ### along with Octave; see the file COPYING.  If not, see
 ### <http://www.gnu.org/licenses/>.
 
 AC_PREREQ([2.63])
-AC_INIT([GNU Octave], [4.0.1-rc3], [http://octave.org/bugs.html], [octave])
+AC_INIT([GNU Octave], [4.0.1-rc4], [http://octave.org/bugs.html], [octave])
 
 dnl Note that the version number is duplicated here and in AC_INIT
 dnl because AC_INIT requires it to be static, not computed from
 dnl shell variables.
 OCTAVE_MAJOR_VERSION=4
 OCTAVE_MINOR_VERSION=0
-OCTAVE_PATCH_VERSION=1-rc3
+OCTAVE_PATCH_VERSION=1-rc4
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION arg
 OCTAVE_VERSION="$PACKAGE_VERSION"
 
 OCTAVE_COPYRIGHT="Copyright (C) 2015 John W. Eaton and others."
 
 OCTAVE_RELEASE_DATE="2016-01-18"
 
